direct/io-kafka/src/main/java/cz/o2/proxima/direct/kafka/KafkaAccessor.java

```diff
@@ -215,6 +232,44 @@ Properties createProps() {
         getUri(), createProps(), serializer.keySerde(), serializer.valueSerde());
   }
 
+  /**
+   * Checker for kafka topics configuration
+   *
+   * @param familyDescriptor Attribute family descriptor.
+   * @return true if check succeeded, otherwise false
+   */
+  @Override
+  public boolean isAcceptable(AttributeFamilyDescriptor familyDescriptor)
+      throws RuntimeException {
+    // Force checks for data compacting on state-commit-log topics
+    if (familyDescriptor.getAccess().isStateCommitLog()) {
+      try (AdminClient admin = createAdmin()) {
+        Collection<ConfigResource> cr =
+            Collections.singleton(new ConfigResource(ConfigResource.Type.TOPIC, this.topic));
+
+        DescribeConfigsResult configsResult = admin.describeConfigs(cr);
+
+        Config config = Iterables.getOnlyElement(configsResult.all().get().values());
+
+        for (ConfigEntry currentCfgEntry : config.entries()) {
```

Comment:
is it possible to perform a lookup in O(1)? Such as `config.get(TopicConfig.CLEANUP_POLICY_CONFIG)`?

Reply:
Yes it is, changed